pom.xml

```diff
@@ -25,115 +23,245 @@
   </licenses>
 
   <properties>
-    <!-- Maven related -->
+
+    <!-- Maven -->
+    <maven.compiler.parameters>true</maven.compiler.parameters>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <version.org.apache.maven.plugins.maven-compiler-plugin>3.10.1</version.org.apache.maven.plugins.maven-compiler-plugin>
-    <version.org.apache.maven.plugins.maven-antrun-plugin>1.8</version.org.apache.maven.plugins.maven-antrun-plugin>
-    <version.org.apache.maven.plugins.maven-failsafe-plugin>2.22.2</version.org.apache.maven.plugins.maven-failsafe-plugin>
-    <version.org.apache.maven.plugins.maven-surefire-plugin>2.22.2</version.org.apache.maven.plugins.maven-surefire-plugin>
-    <version.org.apache.maven.plugins.maven-jar-plugin>3.2.2</version.org.apache.maven.plugins.maven-jar-plugin>
-    <version.org.apache.maven.plugins.maven-resources-plugin>3.0.2</version.org.apache.maven.plugins.maven-resources-plugin>
-    <version.org.apache.maven.plugins.maven-compiler-plugin>3.8.1</version.org.apache.maven.plugins.maven-compiler-plugin>
-    <version.org.codehaus.mojo.exec-maven-plugin>3.0.0</version.org.codehaus.mojo.exec-maven-plugin>
-
-    <!-- Rest of them -->
-    <version.junit>4.13.2</version.junit> <!-- External module uses Quarkus' JUnit 5 -->
-    <version.org.antlr>4.10.1</version.org.antlr>
-    <version.org.apache.commons.commons-math3>3.6.1</version.org.apache.commons.commons-math3>
-    <version.com.google.guava>31.1-jre</version.com.google.guava>
-
-    <!-- Used for openapi generation and itests -->
-    <version.org.codehaus.groovy>3.0.10</version.org.codehaus.groovy>
-    <version.org.codehaus.gmavenplus>1.13.1</version.org.codehaus.gmavenplus>
-    <version.org.codehaus.groovy.modules.http-builder>0.7.1</version.org.codehaus.groovy.modules.http-builder>
 
-    <!-- Quarkus -->
-    <quarkus.version>2.9.1.Final</quarkus.version>
+    <!-- Plugins -->
+    <compiler-plugin.version>3.10.1</compiler-plugin.version>
+    <surefire-plugin.version>3.0.0-M6</surefire-plugin.version>
 
-    <!--Distributed tracing support for Vert.x which is not covered by SmallRye-OT -->
-    <version.opentracing-vertx-web>1.0.0</version.opentracing-vertx-web>
+    <!-- Quarkus -->
+    <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
+    <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
```

Comment:
maybe a stupid question, but does the groupId and artifactId change that often in Quarkus so we need it to be replaceable?

Reply:
Even if we needed to change that (which doesn't happen often), we could still do it in the dependency declaration. All Quarkus projects are generated that way but these properties are a bit useless indeed.

Reply:
then let's just keep it. I was just interested in the reason why we/quarkus is doing this